vital/bindings/python/vital/types/feature.cxx

```diff
@@ -37,6 +37,11 @@
 
 namespace py=pybind11;
 
+// TODO
+// We don't actually need all this extra crap, I wrote this class
+// when trying to figure out how to deal with templates in pybind11
+// Use one of the other (better!) bindings as a guide to redo.
+// When we redo this, make sure to update feature_track_state to match
```

Comment:
This TODO is my main concern with this branch (left over from David H.); maybe this entire class can be deleted?